aea/aea.py

```diff
@@ -36,7 +36,7 @@ class AEA(Agent):
     def __init__(self, name: str,
                  mailbox: MailBox,
                  private_key_pem_path: Optional[str] = None,
-                 timeout: float = 1.0,  # TODO we might want to set this to 0 for the aea and let the skills take care of slowing things down on a skill level
+                 timeout: float = 0.0,  # TODO we might want to set this to 0 for the aea and let the skills take care of slowing things down on a skill level
```

Comment:
I am setting this to 0.0 now as the default. We can then slow things down where we want to rather than globally.